app/controllers/curation_concerns/monographs_controller.rb

```diff
@@ -1,5 +1,10 @@
 class CurationConcerns::MonographsController < ApplicationController
   include CurationConcerns::CurationConcernController
   set_curation_concern_type Monograph
+  # Use the Monograph Presenter
+  def show_presenter
```

Comment:
I did to a force push, but I'm on my work machine now with this branched merged locally. I'll close this PR and issue a new one. Sorry.